src/api/03-user-experience/08-picture-in-picture.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Picture-in-Picture
+description: |
+  Play your videos in a detached floating window that stays on top of other applications using the Picture-in-Picture API.
+date: January 31st, 2022
+length: '1:04'
+video-sources:
+  - src: https://storage.googleapis.com/kino-assets/picture-in-picture/manifest.mpd
+    type: application/dash+xml
+  - src: https://storage.googleapis.com/kino-assets/picture-in-picture/master.m3u8
+    type: application/x-mpegURL
+video-subtitles:
+  - default: true
+    kind: captions
+    label: English
+    src: https://storage.googleapis.com/kino-assets/picture-in-picture/cap-en.vtt
+    srclang: en
+  - default: false
+    kind: captions
+    label: Česky
+    src: https://storage.googleapis.com/kino-assets/picture-in-picture/cap-cs.vtt
+    srclang: cz
+thumbnail: https://storage.googleapis.com/kino-assets/picture-in-picture/thumbnail.png
+media-session-artwork:
+  - sizes: 96x96
+    src: https://storage.googleapis.com/kino-assets/picture-in-picture/artwork-96x96.png
+    type: image/png
+  - sizes: 128x128
+    src: https://storage.googleapis.com/kino-assets/picture-in-picture/artwork-128x128.png
+    type: image/png
+  - sizes: 192x192
+    src: https://storage.googleapis.com/kino-assets/picture-in-picture/artwork-192x192.png
+    type: image/png
+  - sizes: 256x256
+    src: https://storage.googleapis.com/kino-assets/picture-in-picture/artwork-256x256.png
+    type: image/png
+  - sizes: 384x384
+    src: https://storage.googleapis.com/kino-assets/picture-in-picture/artwork-384x384.png
+    type: image/png
+  - sizes: 512x512
+    src: https://storage.googleapis.com/kino-assets/picture-in-picture/artwork-512x512.png
+    type: image/png
+pip: true
+---
+
+## The basics
+
+The three main pieces the Picture-in-Picture API introduces are:
+
+1. The `requestPictureInPicture()` method of `<video>` elements.
+2. The `exitPictureInPicture()` method of the `document`.
+3. The `pictureInPictureElement` property of the `document` or a `shadowRoot`.
+
+A minimal example of controlling the video's Picture-in-Picture state would only consist of calling `requestPictureInPicture` and `exitPictureInPicture` respectively:
+
+```html
+<video src="video.mp4"></video>
+<button>Toggle Picture-in-Picture</button>
+
+<script>
+  const video = document.querySelector('video');
+  const button = document.querySelector('button');
+
+  button.addEventListener('click', () => {
+    document.pictureInPictureElement === video
+      ? document.exitPictureInPicture()
+      : video.requestPictureInPicture();
+  });
+</script>
+```
+
+## Checking for feature support
+
+[Picture-in-Picture API support] is spotty. Even in browsers that implement it, the feature may be disabled by the `picture-in-picture` Feature Policy. There is a handy `pictureInPictureEnabled` property that will be present on the `document` in browsers that implement the API and its value will be `true` if Picture-in-Picture mode is allowed.
```

Comment:
replace `is spotty` with something like `is different from what you're probably used to.`

Reply:
Updated in 281290836b44cdc445dea0ef7410d1082e9d8506.